chia/full_node/mempool_check_conditions.py

```diff
@@ -93,13 +93,13 @@ def get_name_puzzle_conditions(
         err, result = run_block(bytes(generator.program), block_args, max_cost, flags)
         assert (err is None) != (result is None)
         if err is not None:
-            return NPCResult(uint16(err), None, uint64(0))
+            return NPCResult(uint16(err), None)
         else:
             assert result is not None
-            return NPCResult(None, result, uint64(result.cost))
```

Comment:
as can be seen here, the `cost` field is set to `result.cost`, duplicating the value